packages/webapp/components/Governance/Delegation/DelegateButton.tsx

```diff
@@ -12,10 +12,7 @@ const InnerDelegateButton: FC<DelegateButtonProps> = ({ ...props }) => {
   const { data: delegation } = useDelegationInfo(address)
   const { isOpen, onOpen, onClose } = useDisclosure()
   if (!address || !delegation || !delegation?.delegate || !delegation?.account)
-    return <></>
-  const hasDelegatedToAnother =
```

Comment:
assuming this is removed because it's causing issues because delegations aren't implemented yet. if that's the case, i'd advocate for commenting this section out with a TODO so as not to lose the context of the previous implementation

Reply:
Actually I removed it because the variable was not even being used, but `delegation?.account &&
    delegation.account?.delegate.id.toLowerCase() !== address?.toLowerCase()` was causing an error with my ts-lint